registry/application/services/organization_publisher_service.py

```diff
@@ -15,9 +15,11 @@
 from registry.domain.factory.organization_factory import OrganizationFactory
 from registry.domain.models.organization import Organization
 from registry.domain.services.registry_blockchain_util import RegistryBlockChainUtil
+from registry.exceptions import InvalidOrganizationStateException
```

Comment:
test cases are not added or updated?

Reply:
mail functionality was already there, and it is mocked.

Reply:
@pratik-vii You have other changes which need test cases.

Reply:
Call to the notification service is mocked, but testing which template you are choosing based on state and what all recipients are getting picked up can be a good test case.